stylecop.ruleset

```diff
@@ -30,5 +30,9 @@
     <Rule Id="SA1602" Action="None" />
     <Rule Id="SA1649" Action="None" />
     <Rule Id="SA1652" Action="None" />
+    <Rule Id="SA1414" Action="None" />
+    <Rule Id="SA1316" Action="None" />
+    <Rule Id="SA1142" Action="None" />
+    <Rule Id="SA1135" Action="None" />
```

Comment:
SA1135: Bug in stylecop analyzers throwing null ref exception. Bug logged in their repo.

Others: These checks were added when moving to dotnet 3. We weren't following these conventions in dotnet2, so to stick with the same standards I am removing.